datahub-actions/scripts/install_deps.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+set -euxo pipefail
+
+# Copyright 2021 Acryl Data, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+if [ "$(uname)" == "Darwin" ]; then
+    brew install librdkafka
+else
+    sudo apt-get update && sudo apt-get install -y \
+        librdkafka-dev \
+        python3-ldap \
+        libldap2-dev \
+        libsasl2-dev \
+        ldap-utils
```

Comment:
we can reuse the script in metadata-ingestion/scripts/... - no need to duplicate it